larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/dto/TextPIIEntity.java

```diff
@@ -36,7 +36,11 @@
     this.end = end;
   }
 
-  public static TextPIIEntity of(String entityType, double score, int start, int end) {
+  public static TextPIIEntity immutableOf(String entityType, double score, int start, int end) {
+    return new TextPIIEntity(entityType, StarlarkFloat.of(score), StarlarkInt.of(start), StarlarkInt.of(end), null);
```

Comment:
return `of(entityType, StarlarkFloat.of(score), StarlarkInt.of(start), StarlarkInt.of(end), null);`